### PR TITLE
Merge with 'develop', update field errors to bootstrap 4

### PR DESCRIPTION
**Pull request by jonashaag**

### CODE REVIEW

**Comment by jonashaag** [comment]
@dyve merged with latest bootstrap 3 develop

File: bootstrap4/tests.py [hunk 1]
```diff
@@ -260,6 +278,17 @@ def test_required_class(self):
...
+    def test_settings_filter(self):
+        res = render_template_with_form('{{ "required_css_class"|bootstrap_setting }}')
+        self.assertEqual(res.strip(), 'bootstrap3-req')
+        res = render_template_with_form('{% if "javascript_in_head"|bootstrap_setting %}head{% else %}body{% endif %}')
+        self.assertEqual(res.strip(), 'head')
+
+    def test_required_class(self):
+        form = TestForm()
+        res = render_template_with_form('{% bootstrap_form form %}', {'form': form})
+        self.assertIn('bootstrap3-req', res)
```
Comment (mcparadip) [inline bootstrap4/tests.py]: Should be bootstrap4

File: bootstrap4/tests.py [hunk 2]
```diff
@@ -260,6 +278,17 @@ def test_required_class(self):
         res = render_template_with_form('{% bootstrap_form form %}', {'form': form})
         self.assertIn('bootstrap4-req', res)
 
+    def test_settings_filter(self):
+        res = render_template_with_form('{{ "required_css_class"|bootstrap_setting }}')
+        self.assertEqual(res.strip(), 'bootstrap3-req')
```
Comment (mcparadip) [inline bootstrap4/tests.py]: Should be bootstrap4

File: bootstrap4/templatetags/bootstrap4.py [hunk 3]
```diff
@@ -341,7 +341,8 @@ def bootstrap_formset_errors(*args, **kwargs):
             The formset that is being rendered
 
         layout
-            Context value that is available in the template ``bootstrap4/form_errors.html`` as ``layout``.
+            Context value that is available in the template ``bootstrap3/form_errors.html``
```
Comment (mcparadip) [inline bootstrap4/templatetags/bootstrap4.py]: Should be bootstrap4

Comment (mcparadip) [inline bootstrap4/templatetags/bootstrap4.py]: Should be bootstrap3

**Comment by dyve** [comment]
Sorry for the late reply. This PR doesn't pass CI, any chance you could get it up to date?

**Comment by mcparadip** [comment]
I replied with how to fix the issues.

**Pull request closed without merging**